app/views/merchant_items/index.html.erb

```diff
@@ -3,8 +3,18 @@
 
 <ul>
   <% @items.each do |item| %>
-    <div id="item_<%= item.id %>">
-      <h3><li><%= link_to "#{item.name}", merchant_item_path(@merchant.id, item.id) %></li></h3>
-    </div>
+    <% if item.status == true %>
+      <div id="item_<%= item.id %>">
+        <p><li><%= link_to "#{item.name}", merchant_item_path(@merchant.id, item.id) %>
+              <%= button_to "Disable", edit_merchant_item_path(@merchant.id, item.id), method: :get, params:[status: false]%>
```

Comment:
This button can just have a :patch method and instead be directed to the merchant_item_path

Reply:
complete